topics/maven/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: apache-maven
+created_by: Apache Software Foundation
+display_name: "Maven"
+github_url: https://github.com/apache/maven
+logo: maven.png
+released: July 13, 2004
+short_description: "Maven is a build automation tool used primarily for Java projects."
```

Comment:
Does this need the quotes?

Reply:
Good catch on these, it shouldn't need the quotes. Updated!